modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/LifecycleAwareConfigurationProvider.java

```diff
@@ -56,8 +58,7 @@
   private final ConfigurationModel configurationModel;
   private final List<ConfigurationInstance> configurationInstances = new LinkedList<>();
   private final ClassLoader extensionClassLoader;
-  protected SimpleLifecycleManager lifecycleManager =
-      new DefaultLifecycleManager<>(String.format("%s-%s", getClass().getName(), getName()), this);
+  protected final SimpleLifecycleManager lifecycleManager;
```

Comment:
when this was being executed, the name was not set still and the string was ended with `-null`